packages/core/test/app/db/range-bucket.spec.ts

```diff
@@ -45,6 +45,22 @@ const testPutResults = async (
   db: RangeBucket,
   putContents: any[],
   expectedResults: any[]
+): Promise<void> => {
+  // First put the ranges
+  putRanges(db, putContents)
+  // Now check that they were added correctly
+  const res = await db.get(
+    new BigNum('0', 'hex'),
```

Comment:
Can use `ZERO`

Reply:
True, though there are a lot of `'0'`s which I'd have to change in this file. Do you think it's worth it to define a special `ZERO` constant?